tendrl/commons/jobs/__init__.py

```diff
@@ -126,14 +126,21 @@ def _run(self):
                             Event(
                                 ExceptionMessage(
                                     priority="error",
-                                    job_id=job.job_id,
```

Comment:
if job_id and flow_id is present then it will treat as job_update only not as event